cmd/nerdctl/stats.go

```diff
@@ -476,3 +480,15 @@ func statsShellComplete(cmd *cobra.Command, args []string, toComplete string) ([
 	}
 	return shellCompleteContainerNames(cmd, statusFilterFn)
 }
+
+func validateFormatFlag(format string) error {
+	switch format {
+	case "", "table":
+		return nil
+	case "raw":
+		return errors.New("unsupported format: \"raw\"")
+	default:
+		_, err := parseTemplate(format)
```

Comment:
No need to discard the first value

Reply:
This is just validation and hence the first value `tmpl` is not required.
Since, we are already setting and checking again in the ticker, it should be okay. LMK what you think.

Ideally, I would like to avoid allocations every singletime ticker is called; but I was not really sure how to do that. Please let me know if this sounds good to you.

Reply:
Why parse the template on every tick?

Reply:
This was the code before. Let me think and see if I can make it better

Reply:
Can you check this again please?